CameraAnchor.gd

```diff
@@ -1,34 +1,66 @@
 extends KinematicBody2D
 
 var velocity = Vector2(0, 0)
+var viewport = 0
+var margin_w = 0
+var margin_h = 0
+var move = Vector2(0, 0)
+var mouse_right_click_origin = Vector2(0, 0)
+var mouse_position = Vector2(0, 0)
+
+#Should move one screen width every 4 seconds
+#var move_speed = viewport.size.x / 4
+var move_speed = 16834 # just manually setting it idk
```

Comment:
is there any reason that you have it set manually?

Reply:
I couldn't get `viewport.size.x / 4` to work.